File: app/src/main/java/com/example/sena/InicioSesion.java
package com.example.sena;

import android.content.Intent;
import android.os.Bundle;
import android.widget.EditText;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

import com.google.android.material.button.MaterialButton;

public class InicioSesion extends AppCompatActivity {

    private EditText etUsuarioLogin, etContrasenaLogin;
    private MaterialButton btnEntrar, btnRegresar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inicio_sesion);

        etUsuarioLogin = findViewById(R.id.etUsuarioLogin);
        etContrasenaLogin = findViewById(R.id.etContrasenaLogin);
        btnEntrar = findViewById(R.id.btnEntrar);
        btnRegresar = findViewById(R.id.btnRegresar);

        btnRegresar.setOnClickListener(v -> {
            startActivity(new Intent(InicioSesion.this, MainActivity.class));
            finish();
        });

        btnEntrar.setOnClickListener(v -> {
            String usuario = etUsuarioLogin.getText().toString().trim();
            String pass = etContrasenaLogin.getText().toString();

            int res = Administracion.login(this, usuario, pass);

            if (res == 3) {
                Toast.makeText(this, "Completa usuario y contraseña.", Toast.LENGTH_SHORT).show();
                return;
            }
            if (res == 1) {
                Toast.makeText(this, "Ese usuario no existe. Regístrate primero.", Toast.LENGTH_SHORT).show();
                return;
            }
            if (res == 2) {
                Toast.makeText(this, "Contraseña incorrecta.", Toast.LENGTH_SHORT).show();
                return;
            }

            // ✅ IMPORTANTE: guardar sesión actual
            SessionManager.setLoggedUser(this, usuario);

            Toast.makeText(this, "Bienvenido " + usuario, Toast.LENGTH_SHORT).show();
            startActivity(new Intent(InicioSesion.this, Inicio.class));
            finish();
        });
    }
}

File: app/src/main/java/com/example/sena/InicioSesion.kt
package com.example.sena

import android.content.Intent
import android.os.Bundle
import android.widget.EditText
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.google.android.material.button.MaterialButton

class InicioSesion : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_inicio_sesion)

        val btnRegresar = findViewById<MaterialButton>(R.id.btnRegresar)
        val btnEntrar = findViewById<MaterialButton>(R.id.btnEntrar)
        val etUsuarioLogin = findViewById<EditText>(R.id.etUsuarioLogin)
        val etContrasenaLogin = findViewById<EditText>(R.id.etContrasenaLogin)

        btnRegresar.setOnClickListener {
            // vuelve a MainActivity
            finish()
        }

        btnEntrar.setOnClickListener{
            val Usuario = etUsuarioLogin.text.toString().trim()
            val Contrasena = etContrasenaLogin.text.toString().trim()

            if (Usuario.isEmpty() || Contrasena.isEmpty()) {
                Toast.makeText(this, "Por favor completa todos los campos", Toast.LENGTH_SHORT).show()
            }else {
                // (Aquí iría lógica de guardar los datos en una base de datos, si tuvieras una)

                // Mostrar mensaje y redirigir a Inicio
                Toast.makeText(this, "Registro exitoso. Bienvenido, $Usuario", Toast.LENGTH_SHORT).show()

                val intent = Intent(this, Inicio::class.java)
                startActivity(intent)
                finish() // opcional: cerrar Registro para que no vuelva con back
            }

        }
    }
}
